### PR TITLE
Ud2020 persalys

### PR DESCRIPTION
**Pull request by mbaudin47**
Adds new slides for 2020 User's day.
Add slides from 2019 User's day.

### CODE REVIEW

**Comment by jschueller** [comment]
please move the presentation into the userday2020 subfolder

File: README.rst [hunk 1]
```diff
@@ -31,6 +31,10 @@ OpenTURNS Presentations
   - `Release highlights <https://github.com/openturns/openturns.github.io/blob/master/presentation/master/userday2020relhi.pdf>`_
   - `Probabilistic analyses of a spatial vehicle equipment bay <https://github.com/openturns/presentation/blob/master/userday2020/JU_OT_2020_CNES-Study.pdf>`_
   - `Uncertainties in civil engineering: A FE-model based process for research and applications <https://github.com/openturns/presentation/blob/master/userday2020/kobe_OT_user_day_2020_v01.pdf>`_
+  
+- User day 2019
+
+  - `Overview of recently developed features in Persalys <https://github.com/openturns/presentation/blob/master/userday2019/journee_OT_2019_06_07_Phimeca.pdf>`_
```
Comment (jschueller) [inline README.rst:37]: seems you uploaded a pptx, which github does not read, maybe it could be converted to pdf ?

**Comment by mbaudin47** [comment]
It is already in pdf in the repo. You suggest to convert it into pdf on the Travis machin, e.g. with:
```
$ libreoffice --headless --invisible --convert-to pdf journee_OT_2019_06_07_Phimeca.pptx
```

**Comment by jschueller** [comment]
no, there's no pdf in the userday2019 folder, only the pptx
my convern is that the link will be invalid

**Comment by mbaudin47** [comment]
Arf: a side-effect of the gitignore. I add it asap.

**Comment by jschueller** [comment]
also the link in README to the persalys 2020 presentation is missing

**Pull request merged**